skymp5-server/cpp/server_guest_lib/GetBaseActorValues.h

```diff
@@ -43,9 +46,51 @@ struct BaseActorValues
   }
 };
 
-inline BaseActorValues GetBaseActorValues(uint32_t baseId,
+namespace {
+void SetBaseActorValues(espm::LookupResult& result,
```

Comment:
1. Probably not the best name, suggests like we change something. What do you think about `ExtractBaseActorValues`. It's not ideal too though, but at least it suggests that we set values from some struct
2. `result` can be `const`